src/usr/lib/rsetup/cli/wi-fi.sh

```diff
@@ -3,14 +3,38 @@
 ALLOWED_RCONFIG_FUNC+=("connect_wi-fi")
 
 connect_wi-fi() {
+local i result command
     case $# in
-        1)
+        1|2)
             nmcli radio wifi on
-            nmcli dev wifi connect "$1"
-            ;;
-        2)
-            nmcli radio wifi on
-            nmcli dev wifi connect "$1" password "$2"
+
+            case $# in
+                1)
+                    command="nmcli dev wifi connect $1"
+                    ;;
+                2)
+                    command="nmcli dev wifi connect $1 password $2"
+                    ;;
+            esac
+
+            for ((i = 0; i < 10; i++))
+            do
+                eval "$command"
+
+                result=$?
+
+                if ((result == 0))
+                then
+                    echo "WiFi connection successful"
+                    return
+                else
+                    echo "Failed, trying reconnection"
+                    sleep 1
+                fi
+            done
+
+            echo "Maximum number of tries reached, WiFi connection failed"
```

Comment:
```suggestion
            echo "Wi-Fi failed to connect to '$1" after total of $total_retry retries. Quit." >&2
```
可以给$1也赋值给一个有意义的变量名。错误信息里引用了好几次